Missing alarm subcodes in documentation

https://github.com/Yaskawa-Global/motoros2/actions/runs/6311114508/job/17134469130?pr=158

```
src/ErrorHandling.h:188:5: error: no documentation for '8013[9]' in 'doc/troubleshooting.md'
src/ErrorHandling.h:191:5: error: no documentation for '8013[11]' in 'doc/troubleshooting.md'
```

https://github.com/Yaskawa-Global/motoros2/pull/158

--- a/doc/troubleshooting.md
+++ b/doc/troubleshooting.md
@@ -846,7 +846,7 @@ This must be set to an integer value between `1` and `100` milliseconds.
 
 After correcting the configuration, the [changes will need to be propagated to the Yaskawa controller](../README.md#updating-the-configuration).
 
-### Alarm: 8013[8] or [9]
+### Alarm: 8013[8] - [9]
 
 *Example:*
 
@@ -862,7 +862,7 @@ A maximum of 16 rules may be specified.
 
 After correcting the configuration, the [changes will need to be propagated to the Yaskawa controller](../README.md#updating-the-configuration).
 
-### Alarm: 8013[10] or [11]
+### Alarm: 8013[10] - [11]
 
 *Example:*
 

--- a/doc/troubleshooting.md
+++ b/doc/troubleshooting.md
@@ -846,7 +846,7 @@ This must be set to an integer value between `1` and `100` milliseconds.
 
 After correcting the configuration, the [changes will need to be propagated to the Yaskawa controller](../README.md#updating-the-configuration).
 
-### Alarm: 8013[8] - [9]
+### Alarm: 8013[8 - 9]
 
 *Example:*
 
@@ -862,7 +862,7 @@ A maximum of 16 rules may be specified.
 
 After correcting the configuration, the [changes will need to be propagated to the Yaskawa controller](../README.md#updating-the-configuration).
 
-### Alarm: 8013[10] - [11]
+### Alarm: 8013[10 - 11]
 
 *Example:*
 

